sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/plugin/KotlinSensor.kt

```diff
@@ -236,3 +233,15 @@ fun environment(sensorContext: SensorContext) = Environment(
     getFilesFromProperty(sensorContext.config(), SONAR_JAVA_BINARIES) +
         getFilesFromProperty(sensorContext.config(), SONAR_JAVA_LIBRARIES)
 )
+
+private const val PERFORMANCE_MEASURE_ACTIVATION_PROPERTY = "sonar.kotlin.performance.measure"
+private const val PERFORMANCE_MEASURE_FILE_PATH_PROPERTY = "sonar.kotlin.performance.measure.path"
+private const val PERFORMANCE_MEASURE_DESTINATION_FILE = "sonar.kotlin.performance.measure.json"
```

Comment:
The `_PATH_` property seems to be unused. Can we delete it and move the others to the top of the file?